### DISCUSSION TITLE
What kind of ubuntu are you use for build/release openvscode-server?

### DISCUSSION THREAD

**Discussion by jiungbaek** [gitpod-io/openvscode-server#369] [Q&A] [answered]
Hi

I'am using ubuntu 18.04 for build openvscode-server.

when i use 20.04, terminal in openvscode-server is not work, so i'am using 18.04 for build
(yarn && yarn gulp vscode-reh-web-linux-x64-min)

but if build in 18.04, when i run in ubuntu 16.04, some error log is occur
(error log : /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.22' not found)
(also i can solve this issue for install gcc packages)

but openvscode-server original release is not make error log in 16.04.
(also run by docker)

If i run build in 16.04, any error log doesn't make in 16.04, 18.04, 20.04.

what kind of ubuntu version are you using to build openvscode server?

**Comment by filiptronicek** [comment]
Hi there @jiungbaek :wave:! 

The version we use for building has always been `ubuntu-latest`, which is currently representing Ubuntu 20.04 and has been for a while now. We have recently changed the version which Docker uses to run the server, but not the build process. Hope it helps.

**Reply by jiungbaek** [reply]
Thanks.
Do you have any requirement packages in 20.04?

**Comment by jeanp413** [comment, marked as answer]
Actually it's a little more involved, for native dependencies we use this [docker image](https://github.com/gitpod-io/vscode-linux-build-agent/blob/main/centos7-devtoolset8-x64/Dockerfile). You can take a look at the release github action [here](https://github.com/gitpod-io/openvscode-releases/blob/main/.github/workflows/release.yml)

**Reply by filiptronicek** [reply]
Thanks for noting this 🙏
